solc/CommandLineInterface.h

```diff
@@ -104,6 +104,7 @@ class CommandLineInterface
 	/// @arg _json json string to be written
 	void createJson(std::string const& _fileName, std::string const& _json);
 
+	bool checkMutuallyExclusive(std::string const& _optionA, std::string const& _optionB) const;
```

Comment:
Can't this just be a function in an anonymous namespace?

Reply:
Well, right now it uses `m_args` but we could pass that as parameter, too